### PR TITLE
[New Rule] Azure Active Directory PowerShell Sign-in

### LINKED ISSUE #713: [New Rule] Sign-in via Azure Active Directory PowerShell
## Description
Identifies a sign-in via Azure Active Directory PowerShell. 

## Required Info

- **Eventing Sources:**
`filebeat-*`, `logs-azure.*`

- **Platforms**
`azure`

## Optional Info
- **References:**
https://msrc-blog.microsoft.com/2020/12/13/customer-guidance-on-recent-nation-state-cyber-attacks/
https://docs.microsoft.com/en-us/microsoft-365/enterprise/connect-to-microsoft-365-powershell?view=o365-worldwide

## Syntax
```
event.dataset:azure.signinlogs and azure.signinlogs.properties.app_display_name:"Azure Active Directory PowerShell" and azure.signinlogs.properties.token_issuer_type:AzureAD and event.outcome:success
```

## Example Data

image

### PR DESCRIPTION
**Pull request by bm11100**
## Issues
resolves #713 

## Summary
Identifies a sign-in using the Azure Active Directory PowerShell module. PowerShell for Azure Active Directory allows for managing settings from the command line, which is intended for users who are members of an admin role.

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/azure/initial_access_azure_active_directory_powershell_signon.toml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+risk_score = 21
+rule_id = "a605c51a-73ad-406d-bf3a-f24cc41d5c97"
+severity = "low"
+tags = ["Elastic", "Cloud", "Azure", "Continuous Monitoring", "SecOps", "Identity and Access"]
+type = "query"
+
+query = '''
+event.dataset:azure.signinlogs and
+azure.signinlogs.properties.app_display_name:"Azure Active Directory PowerShell" and
+azure.signinlogs.properties.token_issuer_type:AzureAD and event.outcome:(success or Success)
```
Comment (bm11100) [inline rules/azure/initial_access_azure_active_directory_powershell_signon.toml:38]: > event.outcome:(success or Success)

See https://github.com/elastic/detection-rules/pull/716#discussion_r542672273

File: rules/azure/initial_access_azure_active_directory_powershell_signon.toml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    Sign-ins using PowerShell may be done by a system or network administrator. Verify whether the username, hostname,
+    and/or resource name should be signing into your environment. Sign-ins from unfamiliar users or hosts should be
+    investigated. If known behavior is causing false positives, it can be exempted from the rule.
+    """,
+]
+from = "now-25m"
+index = ["filebeat-*", "logs-azure.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Sign-in via Azure Active Directory PowerShell"
```
Comment (brokensound77) [inline rules/azure/initial_access_azure_active_directory_powershell_signon.toml:23]:
```suggestion
name = "Azure Active Directory PowerShell Sign-in"
```

File: rules/azure/initial_access_azure_active_directory_powershell_signon.toml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+[[rule.threat.technique]]
+id = "T1078"
+name = "Valid Accounts"
+reference = "https://attack.mitre.org/techniques/T1078/"
+[[rule.threat.technique.subtechnique]]
+id = "T1078.004"
+name = "Cloud Accounts"
+reference = "https://attack.mitre.org/techniques/T1078/004/"
+
+
```
Comment (brokensound77) [inline rules/azure/initial_access_azure_active_directory_powershell_signon.toml:53]:
```suggestion
```

**Review by brokensound77 — APPROVED** [review]
A few comments then LGTM 👍

File: rules/azure/initial_access_azure_active_directory_powershell_signon.toml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+risk_score = 21
+rule_id = "a605c51a-73ad-406d-bf3a-f24cc41d5c97"
+severity = "low"
+tags = ["Elastic", "Cloud", "Azure", "Continuous Monitoring", "SecOps", "Identity and Access"]
+type = "query"
+
+query = '''
+event.dataset:azure.signinlogs and
+azure.signinlogs.properties.app_display_name:"Azure Active Directory PowerShell" and
+azure.signinlogs.properties.token_issuer_type:AzureAD and event.outcome:(success or Success)
```
Comment (threat-punter) [inline rules/azure/initial_access_azure_active_directory_powershell_signon.toml:38]:
```suggestion
event.dataset:azure.signinlogs and
  azure.signinlogs.properties.app_display_name:"Azure Active Directory PowerShell" and
  azure.signinlogs.properties.token_issuer_type:AzureAD and event.outcome:(success or Success)
```

**Review by threat-punter — APPROVED** [review]
🧝‍♀️

**Review by dstepanic17 — APPROVED** [review]

File: rules/azure/initial_access_azure_active_directory_powershell_signon.toml (hunk 4 above)
Comment (bm11100) [reply rules/azure/initial_access_azure_active_directory_powershell_signon.toml]: 👍

**Pull request merged**